src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -469,7 +469,13 @@ public bool BackgroundPreviewActive
         {
             get { return BackgroundPreviewViewModel.Active; }
         }
-        
+
+        /// <summary>
+        /// If true, Analytics and Usage options are hidden from UI 
+        /// TO DO: include to StartConfiguration for Dynamo 3.0
+        /// </summary>
+        public bool HideReportOptions { get; set; } = false;
```

Comment:
Is this meant to be public?

Reply:
I need to set it from outside somehow. I would rather make it private and use StartConfiguration, because HideReportOptions is meant to be set only once, before creating the view. But I concerned about doing it in minor release. Do you have any ideas were the best place to put this boolean?

Reply:
@DmytroMuravskyi as long as you do not modify the interface, and only add new members to the concrete class - I believe it should be fine API compatibility wise.

Reply:
@mjkkirschner Agree, sorry that I did not see the view binding. With that, this has to be public

Reply:
@mjkkirschner @QilongTang I've put it into StartConfiguration and made it public and readonly in ViewModel. 
Similar to ShowLogin, you can skip it and default false is used.
I see the PR is already approved, please take a final look at DynamoViewModel.cs